src/test/java/htsjdk/tribble/annotation/StrandTest.java

```diff
@@ -0,0 +1,94 @@
+package htsjdk.tribble.annotation;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+/**
+ * Unit tests for {@link Strand}.
+ * <p>
+ *     Makes sure that {@link Strand} will behave according to contract in run-time.
+ * </p>
+ */
+public class StrandTest {
```

Comment:
oh, you need to derive this class from HtsTestJava or something like that. take a look at one of the other files.